e2e/utils.go

```diff
@@ -1107,6 +1114,14 @@ func validateController(f *framework.Framework, pvcPath, appPath, scPath string)
 	if err != nil {
 		return err
 	}
+	if !checkResize {
```

Comment:
it is easier to validate the resize functionality inside this `if`, you can then also simply move the operations to a helper function (and remove `// nolint:gocyclo` again).